.travis.yml

```diff
@@ -1,20 +1,22 @@
 language: go
 
 go:
-    - 1.9
-    - "1.10"
+    - "1.11"
+    - "1.12"
     - tip
 
+env:
+  - GOLANGCILINT=$(curl -fsSLI -o /dev/null -w %{url_effective} https://github.com/golangci/golangci-lint/releases/latest | awk -F '/' '{print $8}')
+
 go_import_path: github.com/intel/ccloudvm
 
 matrix:
   allow_failures:
   - go: tip
 
 before_install:
-  - go get github.com/alecthomas/gometalinter
-  - gometalinter --install
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin ${GOLANGCILINT}
 
 script:
    - go env
-   - gometalinter --tests --vendor --disable-all --enable=misspell --enable=vet --enable=ineffassign --enable=gofmt --enable=gocyclo --cyclo-over=15 --enable=golint --enable=errcheck --enable=deadcode ./...
+   - golangci-lint run -E gofmt ./...
```

Comment:
I think we're missing gocyclo and mispell.  Not sure how to pass the --cyclo-over=15 parameter

Reply:
@markdryan the default is `30`. If we wish to make it configurable, we can create a yaml file and configure each of those options and pass it to `golangci-lint`. Would that work?

Reply:
Eitherways we will change this command to be `golangci-lint run -E gofmt -E gocyclo -E misspell ./...`

Reply:
>  If we wish to make it configurable, we can create a yaml file and configure each of those options and pass it to golangci-lint. Would that wo

Yes, let's do that.  30 is too high.